From 4d1e8de42a2054265b6969fbcd193c73e45b3067 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Tue, 10 Dec 2024 16:57:11 +0800
Subject: [PATCH 215/274] AOSCOS: MIPS: loongson64: replace CPU_LOONGSON3 with
 CPU_LOONGSON64

Per commit 30ad29bb4888 ("MIPS: Loongson: Naming style cleanup and
rework"), rename the CPU_LOONGSON3 definition as CPU_LOONGSON64.

Fixes: "AOSCOS: MIPS: Loongson 3: Introduce disable_unused_cpus() to save power"
Signed-off-by: Mingcong Bai <jeffbai@aosc.io>

Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 arch/mips/loongson64/pm.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/mips/loongson64/pm.c b/arch/mips/loongson64/pm.c
index c65a39cd7948..8fb7097c4924 100644
--- a/arch/mips/loongson64/pm.c
+++ b/arch/mips/loongson64/pm.c
@@ -35,7 +35,7 @@ static int lefi_pm_enter(suspend_state_t state)
 
 static void lefi_pm_wake(void)
 {
-#ifdef CONFIG_CPU_LOONGSON3
+#ifdef CONFIG_CPU_LOONGSON64
 	disable_unused_cpus();
 #endif
 #ifdef CONFIG_LOONGSON64_CPUAUTOPLUG
-- 
2.47.1

